ssh-pkcs11.c

```diff
@@ -1287,6 +1572,7 @@ pkcs11_fetch_keys(struct pkcs11_provider *p, CK_ULONG slotidx,
 		note_key(p, slotidx, __func__, key);
 		if (pkcs11_key_included(keysp, nkeys, key)) {
 			debug2_f("key already included");;
+			error("freeing already included key %ld", ck_key_type);
```

Comment:
it is not completely clear to me why this is changed here.

This code branch is usually executed when the card has both public key and certificate, which result in the same sshkey and this prevents using duplicate keys.

Reply:
Thanks! Fixed, will be pushed soon